### PR TITLE
da: Run through mdbook-i18n-normalize 0.3.0

### PR DESCRIPTION
**Pull request by mgeisler**
This splits translated code blocks into individual messages, thus preserving the translation after #1460.

This removes ~6k unnecessary lines from the PO file.

### CODE REVIEW

**Comment by mgeisler** [comment]
I had to manually increment the POT-Creation-Date file because of https://github.com/google/mdbook-i18n-helpers/issues/117.

**Comment by mgeisler** [comment]
Hi @thedataking, can you take a peek at this mechanical PR?

**Review by thedataking — APPROVED** [review]
LGTM. Thanks for the nudge and sorry about the late review.

File: po/da.po [hunk 1]
```diff
@@ -1712,27 +1711,17 @@ msgstr ""
...
 
 #: src/cargo/code-samples.md:19
 msgid "You can use "
 msgstr "Du kan bruge "
 
 #: src/cargo/code-samples.md:19
-msgid " to execute the code when focus is in the text box."
-msgstr " for at afvikle koden når fokus er i tekstboksen."
+msgid "to execute the code when focus is in the text box."
+msgstr "for at afvikle koden når fokus er i tekstboksen."
```
[the remark this answers is not in the document]
Comment (mgeisler) [reply po/da.po:1724]: Yeah, that's much better indeed!

```suggestion
msgstr "for at afvikle koden når tekstboksen er i fokus."
```

**Pull request merged**